core/new-gui/src/app/workspace/service/drag-drop/drag-drop.service.ts

```diff
@@ -45,6 +45,12 @@ import * as joint from 'jointjs';
 export class DragDropService {
 
   private static readonly DRAG_DROP_TEMP_OPERATOR_TYPE = 'drag-drop-temp-operator-type';
+  // a subject that can restore the value passed from navigation.component.ts
+  public handleZoomBus: Subject<number> = new Subject<number>();
```

Comment:
Rename this subject to be `workflowEditorZoomSubject` and make this variable `private`.

Another method, `getWorkflowEditorZoomStream()`, should be defined and returned an observable for `workflowEditorZoomSubject`. Then, the components should call `getWorkflowEditorZoomStream.subscribe()` to subscribe for the zoom event.